src/views/Onboarding/GetStarted.vue

```diff
@@ -379,15 +355,16 @@ export default {
   border-radius: 4px;
 }
 
+.welcome {
+  max-width: 640px;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
The same question here. Could we use tailwind's built in classes instead of custom styles? Especially as it's duplicated here

Reply:
https://github.com/kumahq/kuma-gui/pull/179#discussion_r670411039